### DISCUSSION TITLE
whats wrong with my request... i get 404

### DISCUSSION THREAD

**Discussion by elschman** [osm-search/Nominatim#2178] [Q&A]
https://nominatim.openstreetmap.org/search?street=Am+Hafen+19&city=Greifswald&postalcode=17493&format=xml&polygon=0&addressdetails=0&email=osm@valid_domain_here.de&countrycodes=de.
I just need the geo coords ?

What should I do to prevent locked cos of "high" loads ? Adressblock in Excel is around 100 entries.

**Comment by lonvia** [comment]
If you get a 404 then that usually means that you are using a client that we have actively blocked. What client do you use to send the queries? Can you find out what user agent and referrer it sends?

**Reply by elschman** [reply]
Tried to change the agent to some teststuff.. with same result

GET /search?street=Am+Hafen+19&city=Greifswald&postalcode=17493&format=xml&polygon=0&addressdetails=0&email=osm@valid_domain_here.de&countrycodes=de HTTP/1.1
Host: nominatim.openstreetmap.org
User-Agent: test for excel
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Connection: keep-alive
Upgrade-Insecure-Requests: 1
Pragma: no-cache
Cache-Control: no-cache

**Reply by elschman** [reply]
which agent should I use? Especially if this should use the webservice function from excel in future.

**Reply by elschman** [reply]
Now I get 503

**Reply by mtmail** [reply]
503 means the IP got blocked automatically, it will unblock automatically after a couple of hours.

My experience with Excel, Excel in the cloud (office 365) and Google Sheets is they execute formulas asyncronous and in parallel, easily dozens, but I've also seen 100 requests per second on large sheets. Without extra queuing logic it will be hard to stay within 1 request per second limit for the public Nominatim instance https://operations.osmfoundation.org/policies/nominatim/

**Reply by elschman** [reply]
So I've to write some kind of proxy limiting to a slowmotion with less than 1request/s  ... For me ok cos I do need the update may be ones in a year....   Is there any kind of payed solution available which isnt google based. ?

**Reply by lonvia** [reply]
Just make sure the requests are sent sequentially one after another and you will be fine.

I can't find any trace of the requests you describe in the server logs, so I can't really help further. I'd need to find the log of an actual query you have been sending in the last two hours or so.
